seed/challenges/01-front-end-development-certification/intermediate-bonfires.json

```diff
@@ -352,6 +352,7 @@
             "Traduce la cadena de texto que se te provee al Latín de los cerdos (Pig Latin)",
             "<a href=\"http://en.wikipedia.org/wiki/Pig_Latin\" target=\"_blank\">Pig Latin</a> toma la primera consonante (o grupo de consonantes) de una palabra en inglés, la mueve al final de la palabra y agrega un \"ay\".",
```

Comment:
@Arekkusu44 thanks for noticing and letting us know! I've created an [issue](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/10185) in case you'd like to fix it yourself.